packages/utils/src/common/sharding.ts

```diff
@@ -10,3 +20,12 @@ export function ensurePubsubTopicIsConfigured(
     );
   }
 }
+
+//TODO: this can be generalized
```

Comment:
What generalized? 
I would add error handling here: both `JSON.*` and other utils can throw.